xbmc/threads/test/TestHelpers.h

```diff
@@ -76,14 +76,8 @@ class thread
   inline thread(thread& other) : f(other.f), cthread(other.cthread) { other.f = NULL; other.cthread = NULL; }
   inline thread& operator=(thread& other) { f = other.f; other.f = NULL; cthread = other.cthread; other.cthread = NULL; return *this; }
 
-  void join()
-  {
-    cthread->Join(static_cast<unsigned int>(-1));
-  }
+  void join() { cthread->Join(-1ms); }
 
-  bool timed_join(unsigned int millis)
-  {
-    return cthread->Join(millis);
-  }
+  bool timed_join(unsigned int millis) { return cthread->Join(std::chrono::milliseconds(millis)); }
```

Comment:
was this a clang thing? I thought the separate lines was far more readable.

Reply:
Yea it's a clang-format thing